docs: add overview page (#510)
apify/apify-sdk-js: docs/overview.md

---
id: overview
title: Apify SDK for JavaScript
sidebar_label: Overview
---

The Apify SDK for JavaScript is the official library for creating [Apify Actors](https://docs.apify.com/platform/actors) using JavaScript or TypeScript. It provides useful features like actor lifecycle management, local storage emulation, and actor event handling.

> ```javascript

Reviewer: ```suggestion
```js
```

import { Actor } from 'apify';

await Actor.init();

const input = await Actor.getInput();
console.log(input);

await Actor.exit();
```

## What are Actors?

Actors are serverless cloud programs capable of performing tasks in a web browser, similar to what a human can do. These tasks can range from simple operations, such as filling out forms or unsubscribing from services, to complex jobs like scraping and processing large numbers of web pages.

Actors can be executed locally or on the [Apify platform](https://docs.apify.com/platform/), which provides features for running them at scale, monitoring, scheduling, and even publishing and monetizing them.

## Quick start

> To create and run Actors using the Apify Console, see the [Console documentation](https://docs.apify.com/platform/console). For creating and running JavaScript Actors locally, refer to the [Apify platform guide](./guides/apify-platform).

Reviewer: We try to avoid language like `see` and try to stick to `refer` or `check out`. Same goes for directional language, as it does not translate well for screen readers users. 

```suggestion
To create and run Actors using the Apify Console, check out the [Console documentation](/platform/console). For creating and running JavaScript Actors locally, refer to the [Apify platform guide](./guides/apify-platform).
```

Explore the [Examples](./examples) section to see the SDK in action, and refer to the [Guides](./guides) section for a deeper understanding of the SDK's features and best practices.

## Installation

When creating an Actor using the Apify CLI, the Apify SDK for JavaScript is installed automatically. To install it manually in your project, use:

```bash
npm install apify
```

> If your goal is not to develop Apify Actors but to interact with the Apify API from JavaScript, consider using the [Apify API client for JavaScript](https://docs.apify.com/api/client/js) directly.

Reviewer: Why mention CLI all of a sudden when it wasn't mentioned anywhere else in this overview?

```suggestion
## Installation

To install Apify SDK for JavaScript manually in your project, use:

```bash
npm install apify
```

If your goal is not to develop Apify Actors but to interact with the Apify API with JavaScript, consider using the [Apify API client for JavaScript](https://docs.apify.com/api/client/js) directly.
```

Author: I've commited an updated version

---
id: overview
title: Apify SDK for JavaScript
sidebar_label: Overview
---

The Apify SDK for JavaScript is the official library for creating [Apify Actors](https://docs.apify.com/platform/actors) using JavaScript or TypeScript. It provides useful features like Actor lifecycle management, local storage emulation, and Actor event handling.

> ```js

Reviewer 2: I suggest to move this code example to the quick start section
